xl_auth/templates/collections/view.html

```diff
@@ -0,0 +1,93 @@
+<!-- collections/inspect.html -->
+{% extends "layout.html" %}
+{% block content %}
+    <h1>{{ _('View Collection \'%(code)s\'', code=collection.code) }}</h1>
+    <br/>
+    <div class="panel panel-default">
+        <dl class="dl-horizontal">
+            <dt>{{ _('Active') }}:</dt>
+            <dd>{{ _('Yes') if collection.is_active else _('No') }}</dd>
+
+            <dt>{{ _('Replaces') }}:</dt>
+            <dd>
+                {% if collection.replaces %}
+                    <a title="{{ _('View') }}"
+                       href="{{ url_for('collection.view', collection_code=collection.replaces) }}">
+                        {{ collection.replaces }}
+                    </a>
+                {% else %}
+                    -
+                {% endif %}
+            </dd>
+
+            <dt>{{ _('Replaced By') }}:</dt>
+            <dd>
+                {% if collection.replaced_by %}
+                    <a title="{{ _('View') }}"
+                       href="{{ url_for('collection.view', collection_code=collection.replaced_by) }}">
+                        {{ collection.replaced_by }}
+                    </a>
+                {% else %}
+                    -
+                {% endif %}
+            </dd>
+
+            <dt>{{ _('Last Modified') }}:</dt>
+            <dd>
+                {{ collection.modified_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=collection.modified_by.id) }}">
+                    {{ collection.modified_by.full_name }}
+                </a>
+            </dd>
+
+            <dt>{{ _('Created At') }}:</dt>
+            <dd>
+                {{ collection.created_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=collection.created_by.id) }}">
+                    {{ collection.created_by.full_name }}
+                </a>
+            </dd>
+        </dl>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>{{ _('Permissions') }}</strong>
+        </div>
+        <table class="table table-hover">
+            <thead>
+            <tr>
+                <th class="col-md-2">{{ _('User') }}</th>
+                <th class="col-md-3 word-break-all">{{ _('Registrant') }}</th>
+                <th class="col-md-2 word-break-all">{{ _('Cataloger') }}</th>
+                <th class="col-md-3 word-break-all">{{ _('Cataloguing Administrator') }}</th>
+                <th class="col-md-2">{{ _('Created At') }}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for permission in collection.permissions | sort(attribute='created_at') %}
```

Comment:
@mxtthias When listing permissions on a collection, only someone who is a) a cataloging admin on that collection or b) a superadmin, should see permissions for all users. Users that does not fulfill these criterion sees only the collection's cataloging admins (but not themselves).

Reply:
:+1: